app/src/features/projects/view/components/LocationBoundary.tsx

```diff
@@ -18,6 +21,37 @@ const LocationBoundary: React.FC<IProjectDetailsProps> = (props: any) => {
     projectWithDetailsData: { location }
   } = props;
 
+  let geometryCollection: Feature[] = [];
+  let bounds: any[] = [];
+
+  /*
+    Leaflet does not know how to draw Multipolygons so we manually
+    convert to a Feature[] of GeoJSON objects which it can draw
+
+    We also set the bounds based on those geometries so the extent is set
+  */
+  if (location.geometry[0].type === 'MultiPolygon') {
+    location.geometry[0].coordinates.forEach((geoCoords: any) => {
```

Comment:
(Doesnt need to happen now) but does it make sense to move this into a util somewhere, possibly with any other generic map type util functions?

Reply:
mostly thinking of the bit of logic that converts from a multi polygon to a regular feature array*

Reply:
Assuming we think that might be useful elsewhere, but maybe something we can hold off on in case its never used again

Reply:
Dont mind me, just chatting with myself :)